scripts/upgrade

```diff
@@ -95,6 +95,29 @@ pushd $install_dir
 )
 popd
 
+#=================================================
+# BUILD GHOSTCRIPT SPECIFIC VERSION
+#=================================================
+if ! dpkg --compare-versions "$($install_dir/bin/gs --version 2>/dev/null)" ge "10.02.1"
+then
+    ynh_script_progression --message="Building Ghostscript dependency... (this will take a long time!)"
+
+    ynh_setup_source --dest_dir="$install_dir/ghostscript" --source_id="gs" --full_replace
+
+    pushd $install_dir/ghostscript
+        ynh_exec_warn_less ./configure
+        ynh_exec_warn_less make
+        mv bin/ $install_dir/
+    popd
+
+    ynh_secure_remove --file="$install_dir/ghostscript"
+    chmod -R o-rwx "$install_dir/bin"
```

Comment:
other must not have write access, or?

Reply:
It's a bit on principle and mimicking what's already being done on `$install_dir` ;)